app/code/Magento/InventoryAdminUi/README.md

```diff
@@ -1,5 +1,23 @@
-# InventoryAdminUi
+# InventoryAdminUi module
 
-**InventoryAdminUi** provides implementation for inventory management interface.
-See [concept documentation](https://github.com/magento-engcom/magento2/wiki/Technical-Vision.-Catalog-Inventory)
-for further information.
+The `InventoryAdminUi` module extends the Magento Admin UI to add Inventory Management functionality.
+
+This module is part of the MSI (Multi-Source Inventory) project. The 
+[Inventory Management overview](https://devdocs.magento.com/guides/v2.3/inventory/index.html)
+describes the MSI project in more detail.
+
+## Installation details
+
+This module is installed as part of Magento Open Source. You can remove it if you run a headless version of Magento.
```

Comment:
this is not correct, as all the AdminUi modules are dismountable, and could be removed if the system used in a headless way.

It would be good if we would mention that in `AdminUi` modules